Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2679,7 +2690,13 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
     /// - Returns: The best matching storage key if it was found, otherwise `nil`.
     public func identifier(forAssetName name: String, in parent: ResolvedTopicReference) -> String? {
         let bundleIdentifier = parent.bundleIdentifier
-        return assetManagers[bundleIdentifier]?.bestKey(forAssetName: name)
+        if let assetManager = assetManagers[bundleIdentifier] {
+            return assetManager.bestKey(forAssetName: name)
+        } else {
+            return nil != _externalAssetResolvers[bundleIdentifier]?.resolveExternalAsset(named: name, bundleIdentifier: parent.bundleIdentifier)
+                ? name
+                : nil
```

Comment:
Nit: I found this a little confusing to follow.

```suggestion
            if _externalAssetResolvers[bundleIdentifier]?.resolveExternalAsset(named: name, bundleIdentifier: parent.bundleIdentifier) != nil {
                return name
            } else {
                return nil
            }
```